src/Components/List/List.css

```diff
@@ -0,0 +1,11 @@
+.List {
+  padding: 0.5rem 2rem;
+}
+.List input[type='checkbox'] {
+  margin: 20px 20px 20px 0;
+  transform: scale(2);
+}
+.List input:checked + label {
```

Comment:
I love how large the checkboxes are and that there is a strike through and graying out of each item selected!